applications/GeoMechanicsApplication/tests/line_interface_elements/Dirichlet_multi_stage/ProjectParameters_stage1.json

```
@@ -0,0 +1,147 @@
+    "solver_settings": {
+        "solver_type":                        "U_Pw",
+        "model_part_name":                    "PorousDomain",
+        "domain_size":                        2,
+        "model_import_settings":              {
+            "input_type":       "mdpa",
+            "input_filename":   "3+3_line"
+        },
+        "material_import_settings":              {
+            "materials_filename":       "../common/MaterialParameters.json"
+        },
+        "time_stepping":              {
+            "time_step":            1.0
+        },
+        "buffer_size":                        2,
+        "echo_level":                         1,
+        "clear_storage":                      false,
+        "compute_reactions":                  true,
+        "move_mesh_flag":                     false,
+        "reform_dofs_at_each_step":           true,
+        "nodal_smoothing":                    false,
+        "block_builder":                      true,
+        "solution_type":                      "Quasi-Static",
+        "scheme_type":                        "Newmark",
+        "reset_displacements":                false,
+        "newmark_beta":                       0.25,
+        "newmark_gamma":                      0.5,
+        "newmark_theta":                      0.5,
+        "rayleigh_m":                         0.0,
+        "rayleigh_k":                         0.0,
+        "strategy_type":                      "newton_raphson",
+        "convergence_criterion":              "displacement_criterion",
+        "displacement_relative_tolerance":    1.0E-4,
+        "displacement_absolute_tolerance":    1.0E-4,
+        "residual_relative_tolerance":        1.0E-4,
+        "residual_absolute_tolerance":        1.0E-9,
+        "min_iterations":                      6,
+        "max_iterations":                      15,
+        "number_cycles":                      1,
+        "reduction_factor":                   1.0,
+        "increase_factor":                    1.0,
+        "desired_iterations":                 4,
+        "max_radius_factor":                  10.0,
+        "min_radius_factor":                  0.1,
+        "calculate_reactions":                true,
+        "max_line_search_iterations":         5,
+        "first_alpha_value":                  0.5,
+        "second_alpha_value":                 1.0,
+        "min_alpha":                          0.1,
+        "max_alpha":                          2.0,
+        "line_search_tolerance":              0.5,
+        "rotation_dofs":                      false,
+        "linear_solver_settings":             {
+            "solver_type":     "amgcl",
+            "smoother_type":   "ilu0",
+            "krylov_type":     "gmres",
+            "coarsening_type": "aggregation",
+            "max_iteration":   100,
+            "verbosity":       0,
+            "tolerance":       1.0e-6,
+            "scaling":         false
+        },
```

In the end, I used this solver (a bit less of a hassle to set up and should do the same)
```suggestion
        "linear_solver_settings":             {
            "solver_type":     "LinearSolversApplication.sparse_lu"
        }
```

---

Thanks for pointing this out. I have simplified the linear solver settings accordingly in the `ProjectParameters.json` files that have been added in this PR.